src/preferences.ui

```diff
@@ -359,16 +366,18 @@
                             <property name="vexpand">True</property>
                             <property name="row-spacing">6</property>
                             <child>
-                              <!-- n-columns=2 n-rows=1 -->
+                              <!-- n-columns=2 n-rows=2 -->
```

Comment:
With 2 rows as default, how does it look with smaller cpu`s, ie. 2/4 cores/threads ?

Reply:
It's just a comment

Reply:
opps, sorry for that noise.